README.md

```diff
@@ -40,6 +42,13 @@ To migrate a single token to a new bridge, follow the steps below:
 1. Deploy the new token bridge and connect it to the same escrow as the one used by this bridge.
 2. Unregister the token on both `L1TokenBridge` and `L2TokenBridge`, so that no new outbound message can be sent to the other side of the bridge for that token.
 
+## Tests
```

Comment:
We should add the upgradability path as the main Upgrades form ^^^

Reply:
Added. I didn't explicitly say that this is the main/recommended mechanism though, but let me know if you prefer to be more explicit about it.